Example/Storage/Tests/Unit/FIRStorageReferenceTests.m

```diff
@@ -162,4 +162,28 @@ - (void)testCopy {
   XCTAssertNotEqual(ref, copiedRef);
 }
 
+- (void)test_GivenReferenceWithPath_WhenPutNonExistingFile_ThenCompletionIsCalledWithError {
```

Comment:
Although I do think this is a better naming format, it doesn't match the naming for the rest of the file. Maybe something like `testReferenceWithNonExistentFileFailsWithCompletion` would match better?

Reply:
yeah, sounds good, I'll correct this.